puppet.go

```diff
@@ -109,7 +108,7 @@ func ParseConfigFromPuppet(puppetParam, puppetConf string) *GraphConfig {
 	return &config
 }
 
-func PuppetInterval(puppetConf string) chan bool {
+func GetPuppetInterval(puppetConf string) int {
```

Comment:
You don't need to change the name. Golang avoids Get prefixes when possible. You had it right before :)